PSix/AppDelegate.swift

```diff
@@ -16,8 +18,42 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
 
     func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
-        // Override point for customization after application launch.
-        return true
+        // [Optional] Power your app with Local Datastore. For more info, go to
+        // https://parse.com/docs/ios_guide#localdatastore/iOS
+        Parse.enableLocalDatastore()
+        
+        // Initialize Parse.
+        Parse.setApplicationId("3VYFRciY4tMgYwEwVYq2veNGbqq7CjJvQUp85IAo",
+            clientKey: "Ny6h1E3DaoO5M6Kgc8tyJZEu2uMiIr7QnqRe3PqY")
+        PFFacebookUtils.initializeFacebookWithApplicationLaunchOptions([:])
+        
+        // [Optional] Track statistics around application opens.
+        PFAnalytics.trackAppOpenedWithLaunchOptions(launchOptions)
+        
+        window = UIWindow(frame: UIScreen.mainScreen().bounds)
+        let storyboard = UIStoryboard(name: "Main", bundle: nil)
+        let initialViewController: UIViewController
+        
+        if CurrentUser.isLoggedIn() {
+            initialViewController = storyboard.instantiateViewControllerWithIdentifier("OnboardingViewController") as! OnboardingViewController
+        } else {
+            initialViewController = storyboard.instantiateViewControllerWithIdentifier("EventsListViewController") as! EventsListViewController
+        }
+        
+        window?.rootViewController = initialViewController
+        window?.makeKeyAndVisible()
```

Comment:
This is kind of weird.
It's much more common to have an initial storyboard with the initial loading screen set up regularly 
and then in the app delegate or somewhere else global check for user status and if needed modally show the on boarding flow.

Also like Gelber said - I think it'll be nicer to separate the on boarding to it's own storyboard- but that's totally up to you